core/designsystem/src/main/kotlin/com/google/samples/apps/nowinandroid/core/designsystem/component/LoadingWheel.kt

```diff
@@ -54,14 +54,16 @@ import kotlinx.coroutines.launch
 fun NiaLoadingWheel(
     contentDesc: String,
     modifier: Modifier = Modifier,
+    rotationTimeInMillis: Int = 12000,
+    numberOfLines: Int = 12,
```

Comment:
It is better to check the validity of these numbers if you want to customize the value. If you pass 0 or negative numbers it would be nonsense.

Reply:
Maybe let's set a default value if the client pass invalid arguments instead of throwing exceptions.

Reply:
If numberOfLines is too big, it could be performance problem. @IntRange shows warning only and it doesn't enforce the value between the range.